CHANGELOG.md

```diff
@@ -166,7 +166,7 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 
 
-## [21.6.93](https://github.com/WTW-IM/es-components/compare/v21.6.93-pre-prod.5...v21.6.93) (2022-10-19)
+# [21.7.0-pre-prod.1](https://github.com/WTW-IM/es-components/compare/v21.7.0-pre-prod.0...v21.7.0-pre-prod.1) (2022-12-15)
```

Comment:
I don't think we want the changes to this file. (Should only be modified by the merge CI.)

Reply:
That should also be true for all the `CHANGELOG.md` files in these changes.